admin/includes/auto_loaders/config.core.php

```diff
@@ -186,12 +195,12 @@
 /**
  * Breakpoint 100.
  *
- * $messageStack = new messageStack();
+ * $messageStack->add_from_session();
  *
  */
-  $autoLoadConfig[100][] = array('autoType'=>'classInstantiate',
-                                 'className'=>'messageStack',
-                                 'objectName'=>'messageStack');
+  $autoLoadConfig[100][] = array('autoType'=>'objectMethod',
+                                 'object'=>'messageStack',
+                                 'method'=>'add_from_session');

```

Comment:
Turns out there was a bug in this previously-unused feature of the init system, where it wasn't listening properly for these properties.

Fixed in https://github.com/zencart/zencart/commit/2ebd1a1a35940cd483d68daa5a4c775187f006b9

Reply:
If I am following correctly though, then this is a "change" (not necessarily a bad thing) from: https://github.com/zencart/zencart/blob/68b372f5cf95c5cbf666fa6d893897ac606a3c97/includes/autoload_func.php#L49-L54

Won't this mean that others that perhaps have used this method of activating an object's method will have to modify their code to support execution on 1.5.7? (I guess I could see where they just include both sets of keys `'object'` and `'objectName'` as well as `'method'` and `'methodName'` and shouldn't have a problem?!...

Reply:
Do you have an example of where this was ever used that way?

I wouldn't recommend including both, unless you wrap two separate sets of entries in an `if` conditional based on ZC version. That way you can clearly identify dead code when it's time to stop supporting the old.

Reply:
Acckkk.. With a little further historical code review, it looks like the access method is/will be the same, it's just that the "example" has changed...

In ZC 1.5.6 and below the `includes/autoload_func.php` was as below:
https://github.com/zencart/zencart/blob/c80db1cb9b46d08de18c03606713c3050a06d165/includes/autoload_func.php#L83-L93

Messy and all, but clearly identified the need to use `objectName` and `methodName`.  In modifying the LP 100 information, I used the same "process" of determining what to include which was to review the contents of `includes/autoload_func.php` and be sure that the array being used contained the key sought.  Woops, not the case any more.

> Do you have an example of where this was ever used that way?

Not used in *that* way, but the `objectMethod` aspect is/was used:
https://github.com/mc12345678/Stock_By_Attributes_Combined/blob/22d8151bd75fb82961658a70681bfc907d628469/includes/auto_loaders/config.products_with_attributes_stock.php#L38-L42

Reply:
Apologies for the confusion. That temporary refactor in v157 (which uses `object` instead of `objectName`) is (as far as I can tell) an error. And is deprecated.
Now the InitSystem class is used instead, as it also supports the (emerging) plugin stuff:
https://github.com/zencart/zencart/blob/05116c9ffdcd9cb5fd8918b970bd12a19e795885/includes/classes/InitSystem.php#L115-L129